docs/source/changes.rst

```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
+* Add classification support for ``Trainer``. ( `#184`_ )
+
+**Enhance**
+
+* ``BaseFeaturizer.transform`` now supports ``pandas.DataFrame`` as input. (??)
+* Allow using custom elemental information matrix in ``Compositions`` descriptor. ( `#221`_ )
+* Use ``joblib.parallel`` as default parallel backend. ( `#191`_, `#220`_ )
+* ``Splliter.split`` method now support python list as input. ( `#194`_ )
+* Allow user specific index for ``DescriptorHeatmap``. ( `#44`_ )
+* Allow control of number of layers to be extracted in ``FrozenFeaturizer``. ( `#147`_ )
+
```

Comment:
More enhances:
* ``bit_per_entry`` option is added to ``RDKitFP`` and ``AtomPairFP`` to allow control of number of bits to represent one fingerprint entry.
* ``counting`` option is added to ``RDKitFP``, ``AtomPairFP``, ``TopologicalTorsionFP``, ``FCFP`` and ``ECFP`` to support returning counts of each fingerprint entry.
* Column names of ``DescriptorFeature`` is updated to be consistent with the rdkit naming.

Reply:
comments have been added